### DISCUSSION TITLE
Update issue - from v2 to v14

### DISCUSSION THREAD

**Discussion by Chris-win11** [wg-easy/wg-easy#1577] [General]
Hi Dears,

I have issue with updating from v2 to v14. At least I see notification about that. But when I'm doing command in terminal I'm seening that file was downloaded. But why version still old I can't understand. And still docker using password instead of password hash.

![v2-v14](https://github.com/user-attachments/assets/776995d7-d88e-4182-a09e-4b38664b1590)

**1. So i'm removing old containers with this command.**
```
docker stop bf2172e9e283
docker rm bf2172e9e283
```
**Done. It was removed.**

**2. After I'm making update**

docker stop wg-easy
docker rm wg-easy
docker pull ghcr.io/wg-easy/wg-easy:14

Answer from system:

```
root@*********:~# docker stop wg-easy
docker rm wg-easy
docker pull ghcr.io/wg-easy/wg-easy:14
Error response from daemon: No such container: wg-easy
Error response from daemon: No such container: wg-easy
14: Pulling from wg-easy/wg-easy
Digest: sha256:66352ccb4b5095992550aa567df5118a5152b6ed31be34b0a8e118a3c3a35bf5
Status: Image is up to date for ghcr.io/wg-easy/wg-easy:14
ghcr.io/wg-easy/wg-easy:14
```

**So seems file downloaded.**

**3. After I perform docker run**

docker run -d \
  --name=amnezia-wg-easy \
  -e LANGUAGE=en \
  -e WG_HOST=45.83.141.34 \
  -e PASSWORD=4ziB@Arik3 \
  -e PORT=51821 \
  -e WG_PORT=51820 \
  -e CHECK_UPDATE=true \
  -e WG_DEFAULT_DNS=8.8.8.8 \
  -e UI_TRAFFIC_STATS=true \
  -e UI_CHART_TYPE=3 \
  -e USE_GRAVATAR=true \
  -e UI_ENABLE_SORT_CLIENTS=true \
  -e ENABLE_PROMETHEUS_METRICS=true \
  -v ~/.amnezia-wg-easy:/etc/wireguard \
  -p 51820:51820/udp \
  -p 51821:51821/tcp \
  --cap-add=NET_ADMIN \
  --cap-add=SYS_MODULE \
  --sysctl="net.ipv4.conf.all.src_valid_mark=1" \
  --sysctl="net.ipv4.ip_forward=1" \
  --device=/dev/net/tun:/dev/net/tun \
  --restart unless-stopped \
  ghcr.io/spcfox/amnezia-wg-easy

**Answer from system:**
```
846617ce93be0f1c4b3e993f00ba55cce15276d699f2f1ccd3dc29f200974f89
root@vm3378734:~# 
```

Seems everything is fine. But when i open web-gui I see same interface and same notification about updating. Still avatars not available for choose. Statictics showing error

![stat](https://github.com/user-attachments/assets/f8104211-a6dc-4da4-b414-742d64001a6f)
![stat2](https://github.com/user-attachments/assets/59111c8a-2c6b-4dc6-a8d6-59f0abff46cd)

**Comment by kaaax0815** [comment]
v14 doesn't have metrics. only `:nightly` does

you are using `ghcr.io/spcfox/amnezia-wg-easy` which is not the correct project. either switch to wg-easy or file a report on the projects repo
